k-distribution/include/builtin/rat.k

```diff
@@ -111,3 +141,18 @@ module RAT
   rule R <=Rat S => S >=Rat R
 
 endmodule
+
+module RAT-NON-KORE
+  imports RAT
+
+  /*
+   * equalities for non-kore backends such as the java backend
+   */
+
+  rule < I , I' >Rat ==Rat < J , J' >Rat => I ==Int J andBool I' ==Int J'
```

Comment:
as I believe I said elsewhere, this module should be given the attribute [kast] and imported from RAT rather than the other way around.